features/step_definitions/metering.rb

```diff
@@ -167,32 +163,45 @@
     raise "report '#{report_type}' for project '#{namespace}' not found after #{seconds} seconds"
   end
 end
+
 Given /^I enable route for#{OPT_QUOTED} metering service$/ do | metering_name |
   metering_name ||= metering.name
-  htpasswd = BushSlicer::SSL.sha1_htpasswd(username: user.name, password: user.password)
-  cookie_seed = rand_str(32, :hex)
-  route_yaml = <<BASE_TEMPLATE
-    apiVersion: metering.openshift.io/v1alpha1
-    kind: Metering
-    metadata:
-      name: "#{metering.name}"
-    spec:
-      reporting-operator:
-        spec:
-          route:
-            enabled: true
-          authProxy:
-            enabled: true
-            htpasswdData: |
-              #{htpasswd}
-            cookieSeed: "#{cookie_seed}"
-            subjectAccessReviewEnabled: true
-            delegateURLsEnabled: true
+  # create the route only if one does not exists (currently it's hardcoded
+  # in the chart file charts/reporting-operator/values.yaml)
+  unless route('metering').exists?
+    org_user = user
+    if env.is_admin? user
+      # admin user doesn't have name and password properties, we switch to a user
+      step %Q/the first user is cluster-admin/
+      step %Q/I switch to the first user/
+    end
+    htpasswd = BushSlicer::SSL.sha1_htpasswd(username: user.name, password: user.password)
```

Comment:
I think that
1. username and password here don't need to match the username and password of a user
1. you can user1 username and password without making it cluster admin; i.e. `user(0).name`, `user(0).password` but then submit the yaml as the cluster admin user.

Preferable to use fake username and password instead of putting actual user/pass

Reply:
@akostadinov ok, done.  PTAL.  Thanks!